src/v2/view-builder/views/ov/ChallengeOktaVerifyView.js

```diff
@@ -2,16 +2,22 @@ import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
 import AuthenticatorVerifyFooter from '../../components/AuthenticatorVerifyFooter';
 import ChallengeOktaVerifyPushView from './ChallengeOktaVerifyPushView';
 import ChallengeOktaVerifyFastPassView from './ChallengeOktaVerifyFastPassView';
+import NumberChallengePushView from './NumberChallengePushView';
+import { AUTHENTICATOR_METHODS } from '../../../ion/RemediationConstants';
 
 export default BaseAuthenticatorView.extend({
   initialize () {
     BaseAuthenticatorView.prototype.initialize.apply(this, arguments);
-
     const currentAuthenticator = this.options?.appState?.get('currentAuthenticator');
     const selectedMethod = currentAuthenticator?.methods[0];
-    if (selectedMethod?.type === 'push') {
+    if (selectedMethod?.type === AUTHENTICATOR_METHODS.PUSH &&
+      currentAuthenticator.contextualData &&
+      currentAuthenticator.contextualData.correctAnswer) {
+      this.Body = NumberChallengePushView;
+    } else if (selectedMethod?.type === AUTHENTICATOR_METHODS.PUSH) {
       this.Body = ChallengeOktaVerifyPushView;
-    } else {
+    }
+    else {
```

Comment:
style-nit: Did prettier modify this?

Reply:
My editor might have messed the formatting. Fixed!